src/test/jenkins/job/library/libraryJob_dynamic_map.jenkins

```diff
@@ -0,0 +1,10 @@
+def execute() {
+    node() {
+        def lib = library(identifier: 'commons', changelog: false)
+        def utils = lib.net.courtanet.jenkins.Utils.new()
+        sh "${utils.gitTools()}"
+        sh 'sleep 3'
```

Comment:
What’s this sleep about? :)

Reply:
I dunno, I just copy/pasted it from another library test. :see_no_evil: However, let's keep things simple; I'll remove the line (see fixup).